Gemfile

```diff
@@ -86,8 +86,8 @@ group :heroku do
   gem 'puma'
 end
 
-gem 'responds_to_parent', git: 'https://github.com/CloCkWeRX/responds_to_parent.git', branch: 'patch-2' # Temporarily pointed at git until https://github.com/zendesk/responds_to_parent/pull/7 is released
-gem 'acts_as_commentable', git: 'https://github.com/fatfreecrm/acts_as_commentable.git', branch: 'rails-61' # Our fork
+gem 'responds_to_parent'
+gem 'acts_as_commentable'
```

Comment:
Be aware that the upstream maintainer of this gem went MIA.

https://github.com/jackdempsey/acts_as_commentable/compare/master...fatfreecrm:acts_as_commentable:rails-61

I'd recommend forking from our version and making a rails-7 branch